src/Executables/Game/World/SpawnPoint.cs

```diff
@@ -16,6 +16,22 @@ public enum ESpawnPointType
         Special
     }
 
+    /// <summary>
+    /// Spawn point directions follows the compass rose with counter-clockwise increments.
+    /// </summary>
+    public enum ESpawnPointDirection
```

Comment:
```suggestion
    public enum ESpawnPointDirection : byte
```

May save some bytes because this may be used a lot